src/test/java/com/google/jenkins/plugins/storage/IT/ITUtil.java

```diff
@@ -0,0 +1,159 @@
+import com.google.common.io.ByteStreams;
+import com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials;
+import com.google.jenkins.plugins.credentials.oauth.ServiceAccountConfig;
+import com.google.jenkins.plugins.storage.StringJsonServiceAccountConfig;
+import com.google.jenkins.plugins.storage.client.ClientFactory;
+import com.google.jenkins.plugins.storage.client.StorageClient;
+import hudson.AbortException;
+import hudson.EnvVars;
+import hudson.model.ItemGroup;
+import hudson.model.Run;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.UUID;
+import java.util.logging.Logger;
+import org.jvnet.hudson.test.JenkinsRule;
+
+/** Provides a library of utility functions for integration tests. */
+public class ITUtil {
+  private static String projectId = System.getenv("GOOGLE_PROJECT_ID");
+  private static String bucket = System.getenv("GOOGLE_BUCKET");
+
+  /**
+   * Formats a random name using the given prefix.
+   *
+   * @param prefix The prefix to be randomly formatted.
+   * @return The randomly formatted name.
+   */
+  static String formatRandomName(String prefix) {
+    return String.format("%s-%s", prefix, UUID.randomUUID().toString().replace("-", ""));
+  }
+
+  /**
+   * Loads the content of the specified resource.
+   *
+   * @param testClass The test class the resource is being loaded for.
+   * @param name The name of the resource being loaded.
+   * @return The contents of the loaded resource.
+   * @throws IOException If an error occurred during loading.
+   */
+  static String loadResource(Class testClass, String name) throws IOException {
+    return new String(ByteStreams.toByteArray(testClass.getResourceAsStream(name)));
+  }
+
+  /**
+   * Dumps the logs from the specified {@link hudson.model.Run}.
+   *
+   * @param logger The {@link java.util.logging.Logger} to be written to.
+   * @param run The {@link hudson.model.Run} from which the logs will be read.
+   * @throws IOException If an error occurred while dumping the logs.
+   */
+  static void dumpLog(Logger logger, Run<?, ?> run) throws IOException {
+    BufferedReader reader = new BufferedReader(run.getLogReader());
+    String line = null;
+    while ((line = reader.readLine()) != null) {
+      logger.info(line);
+    }
+  }
+
+  /**
+   * Returns the credentialsId, which is the same as the projectId.
+   *
+   * @return the credentialsId
+   */
+  static String getCredentialsId() {
+    return projectId;
+  }
+
+  /**
+   * Delete object matching pattern from Google Cloud Storage bucket of name bucket.
+   *
+   * @param itemGroup A handle to the Jenkins instance.
+   * @param pattern Pattern to match object name to delete from bucket.
+   * @throws IOException If there was an issue making the delete API call to GCS.
+   */
+  static void deleteFromBucket(ItemGroup itemGroup, String pattern) throws IOException {
```

Comment:
These wrappers don't add any value and make the code less straightforward. Please remove and re-factor the call sites to just use the client methods directly.

Reply:
Done